string_lab.js

```diff
@@ -0,0 +1,154 @@
+// JOSEPH P. PASAOA
```

Comment:
Overall good demonstration of string methods and properties along with other elements from previous lessons. Keep up the good work and try to work on hangman project later on.